### PR TITLE
add different install instructions for release and dev

### LINKED ISSUE #217: docs: Remove CRAN installation instructions until it is actually on CRAN
We anticipate that it will be accepted to CRAN, but if it is not then these instructions will only be confusing (especially if we forget about them). 

https://github.com/RMI-PACTA/pacta.multi.loanbook/blob/c0bc24e76f7f7b72d4c3e0bf9458011f0d4f4aa1/vignettes/cookbook.Rmd#L210-L214

Relates to #210

### PR DESCRIPTION
**Pull request by cjyetman**
- closes #217 
- sets us up to do #210 very easily once it's released on CRAN (by simply removing the comment fences)

note: also removes the text saying which repo `install.packages()` installs from, which technically can not be know since it's dependent on the user's setup

note: also removes the text about verifying the install with `library(pacta.multi.loanbook)` because I don't think that makes much sense nor is it really a great way to actually verify install

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 [![R-CMD-check](https://github.com/RMI-PACTA/pacta.multi.loanbook/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/RMI-PACTA/pacta.multi.loanbook/actions/workflows/R-CMD-check.yaml)
 [![CRAN
-status](https://www.r-pkg.org/badges/version/pacta.multi.loanbook)](https://CRAN.R-project.org/package=pacta.multi.loanbook)
+status](https://www.r-pkg.org/badges/version/pacta.multi.loanbook)](https://CRAN.R-project.org/package=pacta.multi.loanbook){.pkgdown-release}
```
Comment (jdhoffa) [inline README.md]: This seems to not render correctly:
Screenshot 2024-11-13 at 19 19 24

**Review by jdhoffa — CHANGES_REQUESTED** [review]
It's hard for me to test if the `::: {.pkgdown-release}`  chunks render without seeing it on `pkgdown`, but I'll take your word!

The `README.md` does not seem to render as expected though.

File: README.md (hunk 1 above)
Comment (jdhoffa) [reply README.md]: @cjyetman pretty sure this needs to be a `README.Rmd` for that to work
😂 your favourite!!

Comment (cjyetman) [reply README.md]: > @cjyetman pretty sure this needs to be a `README.Rmd` for that to work 😂 your favourite!!

hmmm, you might be right... it works on the main page of the pkgdown site, but maybe/probably will be ignored on GitHub's rendering of the README.md

I'll just comment fence it for now

**Comment by cjyetman** [comment]
> It's hard for me to test if the `::: {.pkgdown-release}` chunks render without seeing it on `pkgdown`, but I'll take your word!
> 
> The `README.md` does not seem to render as expected though.

one can run `pkgdown::build_site()` to see the result locally

File: README.md (hunk 1 above)
Comment (cjyetman) [reply README.md]: commented out

**Review by jdhoffa — APPROVED** [review]
🚀

**Review by jacobvjk — APPROVED** [review]

**Pull request merged**